### DISCUSSION TITLE
google keep converter tool crashes after uploading files

### DISCUSSION THREAD

**Discussion by ghost** [standardnotes/forum#2463] [General]

**Comment by atmoio** [comment]
Any chance you can provide a demo file that causes the crash?

**Comment by ghost** [comment]
actually it's the browser (firefox) itself that freezes and stops working, it's weird. I cannot output any crash file

**Comment by atmoio** [comment]
It's probably because the input file is too large. I'd try breaking it up into pieces, by only importing subsets at a time.

**Comment by danarnold** [comment]
I'm getting the tool timing out when I try to upload any file bigger than a few hundred KB. There are a few notes from Keep which include images and are a few MB in size. Uploading even a single one of those causes the tool to hang.

**Comment by JaspalSuri** [comment]
I'm sorry to hear that it can't handle notes with images. I'll pass your feedback along to the dev team!
